frontend/appflowy_flutter/lib/workspace/application/appearance.dart

```diff
@@ -15,6 +15,10 @@ import 'package:google_fonts/google_fonts.dart';
 part 'appearance.freezed.dart';
 
 const _white = Color(0xFFFFFFFF);
+const kDefaultFontFamily = 'Poppins';
+const kDefaultThemeMode = ThemeMode.system;
+const kDefaultThemeName = "Default";
```

Comment:
Where is this `const kDefaultThemeName` used?

Reply:
It's just in this file right now. I couldn't figure out a better place to put it. I was waiting for a reviewer to help out ;)

Reply:
https://github.com/AppFlowy-IO/AppFlowy/pull/3137/commits/fc6516ef7c563f82e0f397a55c459c238e621b9c